x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/syncjob/ConnectorSyncJobConstants.java

```diff
@@ -13,6 +13,7 @@ public class ConnectorSyncJobConstants {
 
     public static final String EMPTY_CONNECTOR_SYNC_JOB_ID_ERROR_MESSAGE =
         "[connector_sync_job_id] of the connector sync job cannot be null or empty.";
+    public static final String EMPTY_WORKER_HOSTNAME_ERROR_MESSAGE = "[worker_host] of the connector sync job cannot be null or empty.";
```

Comment:
Isn't the field called `worker_hostname` ?
```suggestion
    public static final String EMPTY_WORKER_HOSTNAME_ERROR_MESSAGE = "[worker_hostname] of the connector sync job cannot be null or empty.";
```